pipelines/training.py

```diff
@@ -786,44 +793,68 @@ def everything(
 
     if do_ood:
```

Comment:
I think OOD should come after translation, because we need to distinguish between translated commands and translated OODs.